docs/_articles/writing-style.md

```diff
@@ -0,0 +1,15 @@
+---
+title: UI text writing style
+---
+
+The rules are:
+
+- Capitalize first letter of everything - from single words to sentences.
+- Finish with a period only if it's a full sentence - e.g. not if it's a heading, short instruction, etc.
+- Always make punctuation part of the translated string (so that RTL languages don't start with a period, for example).
+- Use real ellipsis character (<code>…</code>) rather than three periods (<code>...</code>)<sup>1</sup>.
+- Remember that our name is capitalized like this: `KoboToolbox`.
+
+If in doubt, please mimick what is already there.
+
+<small><sup>1</sup> In MacOS use `cmd` + `;`. In Windows it's `alt` + `0` `1` `3` `3` on the numberpad. Microsoft Office applications use `ctrl` + `alt` + `.`.</small>
```

Comment:
"Are you sure it's `cmd` and not `alt` (AKA `option`)?", says the non mac user. I guess it varies based on keyboard layout, but I remember using `alt` for special characters (and apparently there's a band named after one such shortcut).
http://hints.macworld.com/article.php?story=20081013021724944

Reply:
> On Linux, press `<compose key>` + `.` followed by `.`, or, if you do not have a compose key configured, press `ctrl` + `shift` + `u`, release, type `2`, `0`, `2`, `6`, and then press `enter`.

Reply:
You're right, I use it so often I didn't even check what keys I press :-D I also added the linux info :)